script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -985,31 +1022,31 @@ Just so you know, I %{i}am%{/i} a princess.
 [sha:029f43c598b10fb8c2d91f25aa478b74818b9fc4]{
 -- Page 568, Offset 16173.
 -- 文句を言うアルクェイドを適当になだめて、改めて地下駐車場を見回す。
-I somehow pacify the complaining Arcueid,
- after which I once again look around the underground parking lot.
+After I somehow manage to pacify Arcueid's petulant grumbling,
+ I take another look around at the underground parking lot.
 }
 [sha:c79b439c8a40af9411466addbe0d636bde0144c7]{
 -- Page 568, Offset 16174.
 -- ……他に吸血鬼がいそうな様子はない。
-It doesn't seem like there are any other vampires around.
+It doesn't seem like there are any other vampires here.
 }
 [sha:c2bb4ee91c4371e4b9ee690c097ab7f7c5082653]{
 -- Page 569, Offset 16175.
 -- ロアとやらが健在ならマンションの住人すべてが襲いかかってきたかもしれないが、今はこれだけだ。
-Had this Roa person still been alive and well, perhaps the whole apartment complex would've come attacking us,
- but for now, this is all there is.
+It's possible that if this Roa person had still been alive and well the entire apartment complex would have swarmed us,
+ but instead it seems like that was all of them.
 }
 [sha:bed2c3852e57fdfb8a9983adfd85d3a63d4dcfc0]{
 -- Page 569, Offset 16176.
 -- 他の住人たちは先輩の言うⅠ階梯からⅡ階梯までの、親基がいなければ死体に戻る下級死徒だったのだろう。
-The other residents were probably those Rank I and II that Senpai were talking about.
- Lower Dead Apostles that go back to just being corpses without a sire.
+The other residents must have been those Rank I and II Dead that Senpai was talking about.
+ Low tier Dead Apostles that go back to just being corpses without a sire.
```

Comment:
Low tier usually is written with a -
Low-tier.

Maybe this shouldn't involve any tiers at all and just be lower ranked, though.